docs/topics/aad.md

```diff
@@ -6,19 +6,81 @@ This walkthrough is to help you get start with Azure Active Directory(AAD) integ
 
 Please also refer to [Azure Active Directory plugin for client authentication](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/README.md) in Kubernetes repo for more details about OpenID Connect and AAD support in upstream.
 
-Note: This feature is not supported on Azure Stack Hub. 
+## Azure Active Directory and Kubernetes
+AAD on Kubernetes allows administrators to give users access to the cluster by using the azure active directory. Users with access to the client group or client service principle will be able to login to the cluster using their Azure credentials and gain access to the cluster. Note however that the user privileges are assigned based on Kubernetes cluster roles. This feature works on clusters deployed on both Azure and Azure Stack Hub. 
 
 ## Prerequisites
 1. An Azure Active Directory tenant, referred to as `AAD Tenant`. You can use the tenant for your Azure subscription;
-2. A `Web app / API` type AAD application, referred to as `Server Application`. This application represents the `apiserver`. For groups to work properly, you'll need to edit the `Server Application` Manifest and set `groupMembershipClaims` to either `All` or `SecurityGroup`.
-3. A `Native` type AAD application, referred to as `Client Application`. This application is for user login via `kubectl`. You'll need to add delegated permission to `Server Application`, please see [troubleshooting](#loginpageerror) section for detail.
+2. Admin access to the Azure Tenant
 
-You also need to delegate permission to the application as follows:
+> [!NOTE]
+> This feature is currently only supported with Active Directories on Azure
```

Comment:
I would mention that the feature is not supported on ADFS, otherwise the note may sound cryptic to some people.